script.js

```diff
@@ -1,11 +1,20 @@
 // Create an array called `colors` that contains three colors.
-
+var colors = ["red", "white", "blue"]{
+}
 // Log the last item in the array to the console.
-
+console.log
 // Create a `for` loop that prints out "My favorite color is [insert color here]" for each item in `colors`.
-
+colors.forEach(function(colors) {
+  console.log("My favorite color is []" + colors)
```

Comment:
👍 the parameter `colors`, may be better called `color`. This is because the function passed to `forEach` is called with _each_ item in the array. `colors` is the array so a good name for each string in there would be `color`.